app/services/ui_notifications/rss_notifications_checker.rb

```diff
@@ -0,0 +1,50 @@
+require 'rss'
+require 'date'
+
+module UINotifications
+  class RssNotificationsChecker
+    def initialize(options = [])
+      @url = options[:url] || 'https://theforeman.org/feed.xml'
```

Comment:
just wondering, are we going to get ssl verification errors ?

Reply:
I don't see any, you can try to run `CreateRssNotifications.perform_now` on the console to verify